pkg/types/conversion/tfjson/tfjson.go

```diff
@@ -147,6 +156,36 @@ func tfJSONBlockTypeToV2Schema(nb *tfjson.SchemaBlockType) *schemav2.Schema { //
 	return v2sch
 }
 
+func schemaV2TypeFromNestedType(ntyp *tfjson.SchemaNestedAttributeType, schema *schemav2.Schema) error {
+	switch ntyp.NestingMode {
```

Comment:
nit: Could you please leave a comment explaining the source of truth for these conversion cases?